worlds/stardew_valley/__init__.py

```diff
@@ -62,15 +65,28 @@ class StardewValleyWorld(World):
     web = StardewWebWorld()
     modified_bundles: Dict[str, Bundle]
     randomized_entrances: Dict[str, str]
+    all_progression_items: Set[str] = set()
```

Comment:
This seems to only be initialized on class, not instance and therefore bleeds through multiple generations/worlds, which doesn't  appear intentional.